packages/core/src/html/includePanelProcessor.js

```diff
@@ -255,7 +255,99 @@ function processInclude(node, context, pageSources, variableProcessor, renderMd,
   return childContext;
 }
 
+/**
+ * PreProcesses popovers with the src attribute.
+ * Replaces it with an error node if the specified src is invalid,
+ * or an empty node if the src is invalid but optional.
+ * Else, sets the content attribute of the popover as parsed from the src.
+ */
+function processPopoverSrc(node, context, pageSources, variableProcessor, renderMd, config) {
+  if (!_.has(node.attribs, 'src')) {
+    return context;
+  }
+
+  if (_.isEmpty(node.attribs.src)) {
+    const error = new Error(`Empty src attribute in include in: ${context.cwf}`);
+    logger.error(error);
+    cheerio(node).replaceWith(createErrorNode(node, error));
+  }
+
+  if (_.has(node.attribs, 'content')) {
+    logger.warn(`${node.name} has a 'src' attribute, 'content' attribute has no effect.`);
+  }
+
+  const {
+    isUrl,
+    hash,
+    filePath,
+    actualFilePath,
+  } = _getSrcFlagsAndFilePaths(node, config);
+
+  const fileExistsNode = _getFileExistsNode(node, context, actualFilePath, pageSources);
+  if (fileExistsNode) {
+    return fileExistsNode;
+  }
+
+  // No need to process url contents
+  if (isUrl) {
+    delete node.attribs.src;
+    return node;
+  }
+
+  pageSources.staticIncludeSrc.push({
+    from: context.cwf,
+    to: actualFilePath,
+  });
+
+  const {
+    nunjucksProcessed,
+    childContext,
+  } = variableProcessor.renderIncludeFile(actualFilePath, pageSources, node, context, filePath);
+
+  let actualContent = nunjucksProcessed;
+  if (fsUtil.isMarkdownFileExt(path.extname(actualFilePath))) {
+    actualContent = renderMd(actualContent);
+  }
+
+  // Process sources with or without hash, retrieving and appending
+  // the appropriate children to a wrapped include element
+  if (hash) {
+    const $ = cheerio.load(actualContent);
+    actualContent = $(hash).html();
+
+    if (actualContent === null) {
+      actualContent = '';
+
+      const error = new Error(`No such segment '${hash}' in file: ${actualFilePath}\n`
+        + `Missing reference in ${context.cwf}`);
+      logger.error(error);
+
+      actualContent = cheerio.html(createErrorNode(node, error));
+    }
+  }
+
+  actualContent = actualContent.trim();
+
+  if (node.children && node.children.length > 0) {
+    childContext.addCwfToCallstack(context.cwf);
+
+    if (childContext.hasExceededMaxCallstackSize()) {
+      const error = new CyclicReferenceError(childContext.callStack);
+      logger.error(error);
+      cheerio(node).replaceWith(createErrorNode(node, error));
+    }
+  }
+
+  const attributeSlotElement = cheerio.parseHTML(`<template #content>${actualContent}</template>`, true);
+  node.children = node.children ? attributeSlotElement.concat(node.children) : attributeSlotElement;
```

Comment:
I think we can abstract this to a function, e.g. `createSlotTemplateNode`, to minimize duplication against a part of `MdAttributeRenderer.processAttributeWithoutOverride`. (Refactoring of the latter function to use the former can be done in a separate work).

I'm not sure that we can put it on `MdAttributeRenderer` too as it doesn't make sense to couple this file to that one, but maybe either on `vueSlotSyntaxProcessor.js` (as it relates to slots), `elements.js` (nice file to put in nodes to use), or any other file you feel is relevant for this.